lib/chef/chef_fs/data_handler/container_data_handler.rb

```diff
@@ -15,10 +15,10 @@ def preserve_key?(key)
           return key == "containername"
         end
 
-        def verify_integrity(object, entry, &on_error)
+        def verify_integrity(object, entry)
```

Comment:
The block argument (alone) is 4x slower than just `yield`ing with no proc:

https://github.com/JuanitoFatas/fast-ruby#proccall-and-block-arguments-vs-yieldcode

Sorta feel like documentation should be fixed with YARD.